src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.BuildOrder.targets

```diff
@@ -25,6 +25,9 @@ projects, these properties are set in Xamarin.Android.Legacy.targets.
       _CheckForObsoleteFrameworkAssemblies;
       _ValidateAndroidPackageProperties;
       AddLibraryJarsToBind;
+      _CollectJavaSourceForBinding;
+      _IncrementalCleanBindings;
+      _CompileBindingJava;
```

Comment:
Are the new targets only for .NET 6? I you'd need some changing in `Xamarin.Android.Legacy.targets` if you want this to also work in Xamarin.Android.

Reply:
This is not available for Legacy.